### ISSUE TITLE
Not working on Fedora 33

### ISSUE THREAD

**Issue by Thesee** [meganz/MEGAsync#627]
Hi, 
just ran a dnf update and then this error show up:
`megasync: symbol lookup error: megasync: undefined symbol: _ZNK8CryptoPP21KeyDerivationFunction16MinDerivedLengthEv`

Installed from mega official repo.
arch x86_64, version: 4.5.3-1.1
other lib versions:
cryptopp 8.6.0-1.fc33
libmediainfo 21.09-1.fc33
libzen 0.4.39-1.fc33

is there any fix?

**Comment by Thesee** [comment]
Update:
last dnf update upgraded cryptopp from 8.2.0-4 to 8.6.0-1 version.
Doing a dnf downgrade cryptopp (rolling back to 8.2.0-4) solved the issue.

So probably there are some breaking changes on the cryptopp libs.

**Comment by polmr** [comment]
There's a new version available in the repository: update your MEGAsync package and the problem shall be gone

**Comment by Thesee** [comment]
@polmr yep, just did an update to megasync-4.5.3-4.1 with cryptopp-8.6.0-1 and it worked.
Thank you.

**Comment by moore-bryan** [comment]
If you're on Fedora, you can downgrade the cryptopp package and "freeze" it by installing `python3-dnf-plugin-versionlock` & running `sudo dnf versionlock add cryptopp` until things are sorted out.

**Comment by moore-bryan** [comment]
For what it's worth—and for anyone who comes along looking for a bandage until this is actually fixed—I only had to versionlock crytpopp and not megasync to get mine working.

**Comment by polmr** [comment]
Updating (refreshing repository metadata):
```
sudo dnf --refresh install megasync
```
should solve the linkage issue and the segfault too, or at least the one here https://github.com/meganz/MEGAsync/issues/629

megasync should be in version  4.5.3-4.1 for Fedora 33 and 4.5.3-3.1 for Fedora 34. Please, ensure that is the one you have installed and that megasync does not list cryptopp as a dynamically linked library. i.e:
```
ldd `which megasync` | grep cryptopp
```
should return empty

**Comment by moore-bryan** [comment]
Interesting... I didn't know that's how packages were handled; that is, if I install directly from the mega site, but have rpmfusion enabled, dnf will try and "marry" them. Thanks, polmr.

Would the "solution" be to disable rpmfusion and reinstall from the rpm? That's not really ideal...

**Comment by polmr** [comment]
Here's what you can do, you just need to exclude megasync from rpm fusion:
```
Edit `/etc/yum.repos.d/rpmfusion-nonfree-updates.repo`
Right before group `[rpmfusion-nonfree-updates-debuginfo]` add `exclude=megasync`
That will leave something like:
```
[rpmfusion-nonfree-updates]
name=RPM Fusion for Fedora $releasever - Nonfree - Updates
#baseurl=http://download1.rpmfusion.org/nonfree/fedora/updates/$releasever/$basearch/
metalink=https://mirrors.rpmfusion.org/metalink?repo=nonfree-fedora-updates-released-$releasever&arch=$basearch
enabled=1
enabled_metadata=1
type=rpm-md
gpgcheck=1
repo_gpgcheck=0
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-rpmfusion-nonfree-fedora-$releasever
exclude=megasync
[rpmfusion-nonfree-updates-debuginfo]
<················>
```
then, you can reinstall:
```
sudo dnf remove megasync && sudo dnf install --refresh --repo=MEGAsync megasync
```
